package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "eduu",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@fontsource/roboto": "^4.5.8",
+    "@fortawesome/fontawesome-free": "^6.4.0",
+    "@fortawesome/fontawesome-svg-core": "^6.4.0",
+    "@fortawesome/free-solid-svg-icons": "^6.4.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "@material-ui/core": "^4.12.4",

```

Comment:
Please remove this package, and execute "npm install". 
You should see "package.json" and "package-lock.json" files in git with changes. You need to commit them and push them **both** after doing the previous steps.

Reply:
I don't need them So I delete them

Reply:
You did not remove "@material-ui/core": "^4.12.4"
This is the one causing the issues, not the ones above it.